genestack_client/cla.py

```diff
@@ -129,21 +129,6 @@ def start(self, accession):
         """
         self.invoke('start', accession)
 
-    # TODO move to filesUtils, why we return name?
-    def rename_file(self, accession, name):
```

Comment:
We need to return this method.  User need to have some time to fix theirs scripts. 

Implementation should be:
```python
# depricated at 25 remove at 26
sys.stderr.write('Depicated ...  use ... instead\n')
FilesUtil(self.connection).rename_file(app_file, name)
```
Current dotorg release is 24, this method will be present while release 25 is on dotorg and will not work after release 26